aries-site/src/layouts/main/Layout.js

```diff
@@ -143,6 +174,38 @@ export const Layout = ({
                 </Box>
               </Main>
             </MainContentWrapper>
+            <FeedbackButton
+              onClick={onOpen}
```

Comment:
Once the feedback layer is opened it feels like the focus should be moved to the layer, currently the focus remains on the button.

Reply:
looking into this 🤔